internal/provider/resource_tfe_team.go

```diff
@@ -141,6 +141,11 @@ func resourceTFETeam() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"allow_member_token_management": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
```

Comment:
Does this need a provider default? The side effect of this choice is that when it is not defined in config, it will be set to true, overriding anything that may be preexisting in the API. Since the API defines its own default, maybe it's best to accept whatever value it returns... Because older TFE may not understand this attribute and will always appear to be false, which (I think) will constantly be a planned change that cannot be changed even if the attribute is not in config.

Reply:
Ah, that's a fair point! Thank you for this feedback! I think it would make sense to go without a default then and rely on the API's value.

Any objections to this, @emlanctot?

Reply:
i agree with @brandonc that there should not be a default on this!